app/services/support_interface/notifications_export.rb

```diff
@@ -0,0 +1,35 @@
+module SupportInterface
+  class NotificationsExport
+    def data_for_export
+      providers = Provider.all.includes(:provider_users, :application_choices)
```

Comment:
This should only return retrieving providers who have open courses in the current recruitment cycle.
To do that you need to check for `open_on_apply` and `recruitment_cycle_year` on the Course model. I have sometime today if you want to pair on this.

Reply:
Sorted, added a condition, thanks for the help!